docs/HowToMonitorWithSumoLogic.md

```diff
@@ -0,0 +1,184 @@
+        [[outputs.sumologic]]
+          url = "https://endpoint1.collection.eu.sumologic.com/receiver/v1/http/<a very very long ascii string goes here>"
+          namepass = [ "Summary.*" ]
+          fieldpass = [ "raw" ]
+          data_format = "carbon2" 
+```
+
+#### Insights graphs on Sumo Logic dashboards
+
+---
+
+Click on Personal -> three dots to the right of the side bar and create a new folder.
+
+![Personal_Folder](./assets/images/sumologic/sumologic21.png)
+
+Click on the 3 dots to the right of the folder name and select Import
+
+![Import](./assets/images/sumologic22.png)
+
+Upload the NuoDB dashboard JSON as a copy paste operation.  The JSON files that can be copy pasted can be found in the following locations
+
+[docs/assets/files/Sumologic_NuoDB_AdHoc.json](assets/files/Sumologic_NuoDB_AdHoc.json)
+
+[docs/assets/files/Sumologic_NuoDB_DBA.json](./assets/files/Sumologic_NuoDB_DBA.json)
+
+[docs/assets/files/Sumologic_NuoDB_NOC.json](./assets/files/Sumologic_NuoDB_NOC.json)
+
+Click "Import" at the bottom of the pane.
+
+![Nuodb_Dashboard_JSON](./assets/images/sumologic25.png)
+
+Set the Collector name and Source name to be the same as set in [Sumo Logic Collectors setup](#sumo-logic-collectors)
+
+![Set_Collector_and_Source](./assets/images/sumologic26.png)
+
+**Note:** The Sumo Logic NuoDB graphs are a subset of the graphs that are available in [Insights](https://github.com/nuodb/nuodb-insights/tree/master). Data that is sent to Sumo Logic is a subset of all the data that is sent to Insights. 
+
+#### Sumo Logic Kubernetes Graphs
+
+---
+
+From the App Catalog search for classic apps "kubernetes" -> "Kubernetes" and follow the install instructions. 
+
+The install instructions look similar to the below:
+
+```
+helm -n <sumologic namespace> upgrade --install sumologic sumologic/sumologic \
+  --namespace=<sumologic namespace> \
+  --create-namespace \
+-f - <<EOF
+sumologic:
+  accessId: <access id>
+  accessKey: <very long access key>
+  clusterName: <Kubernetes clustername>
+  collectorName: <Kubernetes collector name>
+  setup:
+    monitors:
+      enabled: false
+EOF
+```
+
+**Tip:** Use a different namespace other than the namespace for the NuoDB install. 
+
+**Tip:** Take care to make a note of the ```accessId```, ```accessKey```, ```clusterName``` and ```collectorName```.  ```accessKey``` is shown only once.
+
+**Caution:** Some NuoDB logs are duplicated between engine and admin. 
+
+**Tip:** Too much logging? Control [NuoDB logs](https://doc.nuodb.com/nuodb/latest/reference-information/configuration-files/host-properties-nuoadmin.conf/) by option ```logging.consoleLogLevels``` in ```nuoadmin.conf```.
+
+A ```nuoadmin.conf``` example snippet might be: 
+
+```
+"logging.consoleLogLevels": {
+          "ROOT": "error",
+          "DomainProcessStateMachine" : "info",
+          "TagMessageDispatcherRegistry" : "error",
+          "Server" : "error",
+          "org.eclipse.jetty.server.RequestLog" : "error"
+        } ,
```

Comment:
The user does not have direct access to the `nuoadmin.conf` file. They would have to mount in a file that replaces the `nuoadmin.conf.template` that the actual `nuoadmin.conf` file is dynamically created from.

That isn't described here, and I think we should discourage it anyway.  The`nuoadmin.conf.template` shipped in the image is pre-populated with configuration values that make sense in Docker, and we do not want the user messing with it.

An easier and less error prone way to override logging is to use config overrides supplied using the `admin.options` Helm value. E.g. in the Helm values file, you can include:

```
  options:
    ...
    logging.consoleLogLevels: error, DomainProcessStateMachine:info
```

By the way, all the `error` configurations other than the one for the root logger are redundant, since logging configuration is hierarchy. You only need to override the `DomainProcessStateMachine` log level, since that is more verbose than `error`.

Reply:
Changed to 

A example method to add ```logging.consolLogLevels``` for NuoDB [[Helm Charts](https://github.com/nuodb/nuodb-helm-charts/blob/master/stable/admin/README.md#admin)](https://github.com/nuodb/nuodb-helm-charts/blob/master/stable/admin/README.md#admin) using ```admin.options``` Chart option.

Example snippet for admin values yaml.

```
admin:
  options:
    logging.consoleLogLevels: "error,DomainProcessStateMachine:info"
```